pyhuman/app/workflows/browse_web.py

```diff
@@ -23,9 +24,8 @@ def action(self, extra=None):
     def _web_browse(self):
         random_website = self._get_random_website()
         try:
-            with self.driver as d:
-                d.get('https://' + random_website)
-                sleep(2)
+            self.driver.driver.get('https://' + random_website)
+            sleep(2)
```

Comment:
Would it be beneficial to have the sleep timer a configurable parameter?

(if not, define the static value in a global at the top of module)